src/blocks/ImageTextBlock.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react'
+import styled from 'react-emotion'
+import { MaxWidthContainerComponent } from '../components/blockHelpers'
+import {
+  BaseBlockProps,
+  MarkdownHtmlComponent,
+  NativeColorPickerComponent,
+} from './BaseBlockProps'
+
+import {
+  FilledButtonComponent,
+  OutlinedButtonComponent,
+} from '../components/Buttons'
+
+const FilledButtonWithMarginComponent = styled(FilledButtonComponent)({
+  marginTop: '47px',
```

Comment:
imo we should use `rem`s instead of pixels. Thoughts? :) also i think we should avoid these types of "magic numbers" when possible :)

Reply:
i.e. `47px` -> `3rem` :) since 47/16 = ish 3

Reply:
Good points, will do